sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -216,5 +219,37 @@ class VersionsSuite extends SparkFunSuite with Logging {
         "as 'COMPACT' WITH DEFERRED REBUILD")
       client.reset()
     }
+
+    test(s"$version: CREATE TABLE AS SELECT") {
+      withTable("tbl") {
+        sqlContext.sql("CREATE TABLE tbl AS SELECT 1 AS a")
+        assert(sqlContext.table("tbl").collect().toSeq == Seq(Row(1)))
+      }
+    }
+
+    test(s"$version: Delete the temporary staging directory and files after each insert") {
+      import sqlContext.implicits._
```

Comment:
Let us roll back to the way you did in the last run, instead of using the temp table. I am not sure whether this trigger the build issue.